guides/doc-Upgrading_and_Updating/topics/upgrading_process_overview.adoc

```diff
@@ -37,7 +41,14 @@ Upgrading {SmartProxy} takes approximately 10 - 30 minutes.
 
 * Ensure that you have sufficient storage space on your server. For more information, see {InstallingProjectDocURL}#preparing-environment-for-satellite-installation[Preparing your Environment for Installation] in _Installing {ProjectServer} from a Connected Network_ and {InstallingSmartProxyDocURL}preparing-environment-for-capsule-installation[Preparing your Environment for Installation] in _Installing {SmartProxyServer}_.
 
-* Back up your {ProjectServer} and all {SmartProxyServer}s. For more information, see {AdministeringDocURL}backing-up-satellite-server-and-capsule-server[Backing Up {ProjectServer} and {SmartProxyServer}] in the _Administering {ProjectName} {ProductVersionPrevious}_ guide.
+ifeval::["{ProjectVersionPrevious}" == "2.3"]
+* Back up your {ProjectServer} and all {SmartProxyServer}s. For more information, see {SiteURL}/{ProjectVersionPrevious}/Administering_Red_Hat_Satellite/index-foreman.html#backing-up-satellite-server-and-capsule-server
+endif::[]
```

Comment:
i guess this isn't actually  needed since this guide won't exist for 2.3.  TODO: DELETE THIS